### PR TITLE
WFLY-6616 fix clobbering of slave host.xml after we set ignore-unused-configuration="false".

### PR DESCRIPTION
**Pull request by luck3y**
99% of this was Kabir's from  https://github.com/kabir/wildfly/tree/WFLY-6616

### CODE REVIEW

**Comment by luck3y** [comment]
Custom integration run for mixed domain here: https://ci.wildfly.org/viewLog.html?buildId=53748

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

**Comment by luck3y** [comment]
retest this please

File: testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap700/DomainHostExcludes700TestCase.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ *
+ * @author Brian Stansberry
+ */
+@Version(Version.AsVersion.EAP_7_0_0)
+public class DomainHostExcludes700TestCase extends DomainHostExcludesTest {
+
+    @BeforeClass
+    public static void beforeClass() throws InterruptedException, TimeoutException, MgmtOperationException, IOException {
+        LegacyConfig700TestSuite.initializeDomain();
+        setup(DomainHostExcludes700TestCase.class, /*"EAP7.0" Is not in KnownReleases yet */null, ModelVersion.create(4, 1));
```
Comment (bstansberry) [inline testsuite/mixed-domain/src/test/java/org/jboss/as/test/integration/domain/mixed/eap700/DomainHostExcludes700TestCase.java]: /*"EAP7.0" Is not in KnownReleases yet */

^^^ is not true an more.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Looks good other than the one minor comment. Please clone the JIRA to JBEAP and send up a 7.x, and ping me when ready.

**Comment by luck3y** [comment]
@bstansberry Done & JBEAP & 7.x PR sent.

**Pull request merged**